src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -983,6 +994,18 @@ public static bool IsBlank(FormulaValue arg)
             return false;
         }
 
+        public static FormulaValue IsNumeric(EvalVisitor runner, SymbolContext symbolContext, IRContext irContext, FormulaValue[] args)
+        {
+            // Blank or empty. 
+            var arg0 = args[0];
+            return new BooleanValue(irContext, arg0 is NumberValue);
+        }
+
+        public static bool IsNumeric(FormulaValue arg)
```

Comment:
The pattern IsBlank used was because the logic for IsBlank is a little more complex than `a is BlankValue`. Not sure if this IsNumeric function adds any value, especially since it's not actually called from the FormulaValue IsNumeric function.

Reply:
Agree, it's gone.